### PR TITLE
Add Nitro PDF Pro.app v13.2.1

### PR DESCRIPTION
**Pull request by rqelibari**
- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by SMillerDev** [comment]
Please add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap) here. The [CreateZap](https://github.com/nrlquaker/homebrew-createzap) tool can help you with this.

**Comment by rqelibari** [comment]
The tool says `No nitro-pdf-pro settings found`.

File: Casks/nitro-pdf-pro.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  url "https://downloads.gonitro.com/macos/Nitro%20PDF%20Pro%20Retail_#{version}.dmg",
+      verified: "https://downloads.gonitro.com/macos/Nitro%20PDF%20Pro%20Retail_#{version}.dmg"
+  name "Nitro PDF Pro"
+  desc "PDF editing software"
+  homepage "https://pdfpen.com/"
+
+  livecheck do
+    url "https://pdfpen.com/pdfpenpro/download_thanks/"
+    strategy :page_match
```
Comment (bevanjkay) [inline Casks/nitro-pdf-pro.rb:13]:
```suggestion
```
This line can be removed because it is the default strategy.

File: Casks/nitro-pdf-pro.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+cask "nitro-pdf-pro" do
+  version "13.2.1"
+  sha256 "2b59fba1eedec9f2c01c2d1c19544beeca23e7f943f5dcf495674b4466e1a2d2"
+
+  url "https://downloads.gonitro.com/macos/Nitro%20PDF%20Pro%20Retail_#{version}.dmg",
+      verified: "https://downloads.gonitro.com/macos/Nitro%20PDF%20Pro%20Retail_#{version}.dmg"
```
Comment (bevanjkay) [inline Casks/nitro-pdf-pro.rb:6]:
```suggestion
      verified: "downloads.gonitro.com/macos/"
```

**Review by bevanjkay — CHANGES_REQUESTED** [review]

File: Casks/nitro-pdf-pro.rb (hunk 2 above)
Comment (rqelibari) [reply Casks/nitro-pdf-pro.rb]: This shorter URL does not point anywhere. The server responds with a blank page. Shall we use that nonetheless?

**Comment by rqelibari** [comment]
Why is this PR being closed?

**Comment by bevanjkay** [comment]
@rqelibari I'm honestly not sure. It wasn't closed by a user/maintainer, so you may need to reopen in a new PR.

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**